stanford-sw/src/edu/stanford/StanfordIndexer.java

```diff
@@ -592,6 +592,20 @@ MSS MEDIA (e.g. MSS MEDIA 0025), MSS PHOTO (e.g. MSS PHOTO 0463), MSS PRINTS (e.
       }
     }
 
+    /** Based upon SW-1489, if the record if for a certain format (MARC, MRDF,
+     *  MAP, SERIAL, or VM and not SCORE, RECORDING, and MANUSCRIPT) and it has
+     *  something in the 008/28 byte, I’m supposed to give it a genre type of
+     *  government document
+    **/
```

Comment:
I believe all but the SERIAL is included - so I'll add that and replace record if with record is.